### DISCUSSION TITLE
Trivy Azure Compliance Scan Error

### DISCUSSION THREAD

**Discussion by Susanta2000** [aquasecurity/trivy#4905] [Q&A]
### Question

Hi there, I want to run compliance scan in Azure using trivy but it's giving me error. Is trivy support azure?
My command is: trivy k8s cluster --compliance=k8s-nsa --report all
Error is: " 2023-08-01T11:28:32.007+0530 FATAL get k8s artifacts with node info error: running node-collector job: warning event received: Job has reached the specified backoff limit (BackoffLimitExceeded) "

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Version

```bash
0.43.1
```

**Comment by knqyf263** [comment]
@chen-keinan Is it relevant to `--exclude-nodes`?
https://aquasecurity.github.io/trivy/v0.44/docs/target/kubernetes/#infra-checks

**Reply by chen-keinan** [reply]
@knqyf263 don't think so. @Susanta2000 yes it should work on azure.
can you provide more context, are images downloaded from private public registry ?
i there is any proxy or something which might prevent from downloading trivy and node-collector images ?

**Reply by Susanta2000** [reply]
Yes @chen-keinan, images are downloading from public registries. I am seeing some pods statues are "Evicted" and "Error". Does this make any problem?

**Comment by chen-keinan** [comment]
@Susanta2000 I have created issue #4933 , I'll will try to reproduce it
